app/controllers/roles_controller.rb

```diff
@@ -0,0 +1,10 @@
+class RolesController < ApplicationController
+
+  before_action -> { nav_context(:people, :roles) }, except: :accounts
+
+  def index
+    authorize Role
+    @roles = policy_scope(Role)
+    @roles = @roles.sort_by { |x| User::ROLES.index(x) } #this doesn't seem to work
```

Comment:
i actually failed to figure out how to sort these. I think my problem is that @roles isn't an array.